src/Resources/Model.cpp

```diff
@@ -21,8 +21,21 @@ void Model::Load(std::string fileName) {
 
   this->path = fileName;
 
+  // Scale it back up
   logging::Logger::LogDebug("Processing root node");
-  ProcessNode(scene->mRootNode, scene);
+  {
+    auto scale = scene->mRootNode->mTransformation[0][0] == 0.0
```

Comment:
I am not sure what is going on, but whenever we render the models they are not rendered to size. When compared to assimp_view they are much much smaller. More investigation into how these models are supposed to be loaded is needed.